pkg/cmd/cli/config/loader.go

```diff
@@ -12,43 +12,30 @@ const (
 	OpenShiftConfigFlagName        = "config"
 	OpenShiftConfigFileName        = ".openshiftconfig"
 	OpenShiftConfigHomeDir         = ".config/openshift"
-	OpenShiftConfigHomeFileName    = ".config"
+	OpenShiftConfigHomeFileName    = "config"
 	OpenShiftConfigHomeDirFileName = OpenShiftConfigHomeDir + "/" + OpenShiftConfigHomeFileName
-
-	KubeConfigPathEnvVar = clientcmd.RecommendedConfigPathEnvVar
-	KubeConfigFileName   = ".kubeconfig"
-	KubeConfigHomeDir    = ".kube"
 )
 
-// Set up the rules and priorities for loading config files.
-func NewOpenShiftClientConfigLoadingRules() *clientcmd.ClientConfigLoadingRules {
-	return &clientcmd.ClientConfigLoadingRules{Precedence: FullClientConfigFilePriority()}
-}
+var OldRecommendedHomeFile = path.Join(os.Getenv("HOME"), ".config/openshift/.config")
+var RecommendedHomeFile = path.Join(os.Getenv("HOME"), OpenShiftConfigHomeDirFileName)
 
 // File priority loading rules for OpenShift.
-// 1. OPENSHIFTCONFIG env var
-// 2. .openshiftconfig file in current directory
-// 3. ~/.config/openshift/.config file
-func OpenShiftClientConfigFilePriority() []string {
-	return []string{
-		os.Getenv(OpenShiftConfigPathEnvVar),
-		OpenShiftConfigFileName,
-		path.Join(os.Getenv("HOME"), OpenShiftConfigHomeDirFileName),
+// 1. --config value
+// 2. if OPENSHIFTCONFIG env var has a value, use it. Otherwise, ~/.config/openshift/config file
```

Comment:
Missing local .openshiftconfig in this comment. What this code is doing is in fact:

```
// 1. --config value
// 2. Merge ./.openshiftconfig with:
// 2.1. OPENSHIFTCONFIG env var if it has a value, 
// 2.2. otherwise, ~/.config/openshift/config file
```